### ISSUE TITLE
Please update for Big Sur

### ISSUE THREAD

**Issue by maodelg** [MSzturc/Lenovo-T460-OpenCore#21]
Hey, thanks for this. Really clean and flawless work. Will you be willing to upgrade this so we all can have BigSur on the T460? Thanks in advance

**Comment by hylkefaber** [comment]
Hi maodelg please see my repo at https://github.com/hylkefaber/Lenovo-T460-OpenCore for a working T460 with DW1560 Big Sur config. I forgot to remove the '-v' from the boot-args though!

**Comment by aptonline** [comment]
@hylkefaber is this running 0.6.3 of OpenCore?

**Comment by hylkefaber** [comment]
@aptonline yes correct

**Comment by maodelg** [comment]
Thanks @hylkefaber. I was a bit confused by the documentation on your repo, it says Catalina and not mention compatibility with Big Sur. Could you please clarify?

**Comment by hylkefaber** [comment]
That’s correct. I cloned the files from this repo, and changed them where needed for Big Sur compatibility. However, the documentation is still unchanged from this repo, and does not describe the changes I made.

**Comment by hylkefaber** [comment]
I did make a PR for mszturc to include my files and update the documentation. FWIW, I used the Dortania guide to update https://dortania.github.io/OpenCore-Install-Guide/extras/big-sur/

**Comment by aptonline** [comment]
@hylkefaber Are there Big Sur specific changes in your updated EFI? I'm still on Catalina at the moment and just need to update my OpenCore install to the latest so see your repo as a good was to do it.

**Comment by hylkefaber** [comment]
Not really: most changes are related to update OC to 0.6.3. However, OC 0.6.2 is required minimum for Big Sur. Only real Big Sur change is the inclusion of latest AirPortBrcmFixUp kext with updated config, for loading specific sub kext.
